orttraining/orttraining/python/training/artifacts.py

```diff
@@ -0,0 +1,140 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import logging
+import os
+from enum import Enum
+from typing import List, Optional, Union
+
+import onnx
+
+import onnxruntime.training.onnxblock as onnxblock
+
+
+class LossType(Enum):
+    """Enum to represent the loss functions supported by ORT
+
+    To be used with the `loss` parameter of `generate_artifacts` function.
+    """
+
+    MSELoss = 1
+    CrossEntropyLoss = 2
+    BCEWithLogitsLoss = 3
+    L1Loss = 4
+
+
+class OptimType(Enum):
+    """Enum to represent the optimizers supported by ORT
+
+    To be used with the `optimizer` parameter of `generate_artifacts` function.
+    """
+
+    AdamW = 1
```

Comment:
this reminds me I added SGD optimizer but did not add related changes in offline processing yet. :) 
let me add an item track it